CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Changed
+
+- Update dependency versions ([#401])
```

Comment:
nit: Maybe mention _what_ dependencies we are bumping (Rust vs GitHub Actions for example).

Reply:
done